modmesh/pylibmgr.py

```diff
@@ -70,20 +108,10 @@ def search_library_root(curr_path, lib_root_name, timeout=1.0):
             lib_path[item] = os.path.join(_path, item)
 
 
-def load_library(lib_name):
-    """
-    Append library path into python path by user input library name
+def _register_mm_path_finder():
```

Comment:
What about we combine the register function with the existing function `search_library_root()` above?  [`modmesh/core.py#L116`](https://github.com/solvcon/modmesh/blob/7e04e975bc71e137cffd626b865c436f43c0abd3/modmesh/core.py#L116) calls `search_library_root()` already.  In this way, we can remove the global variable `lib_path` too (`sys.meta_path` keeps the meta path created in the combined `search_library_root()`.

Is there a drawback in my proposal?

Reply:
Do you mean that the step of meta path finder registering `sys.meta_path.append(ModmeshPathFinder())` can be put into `search_library_root()` and `lib_path` can be kept in `ModemshPathFinder()` ?

Reply:
Yes.

Reply:
Got it, then the proposal looks to me !
Thanks for your suggestion.